core/src/main/resources/hudson/model/View/newJob.jelly

```diff
@@ -28,14 +28,46 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:s="/lib/form">
   <j:getStatic var="permission" className="hudson.model.Item" field="CREATE"/>
-  <l:layout norefresh="true" permission="${permission}" title="${%NewJob(it.newPronoun)}">
+
+  <l:layout norefresh="true" cssClass="add-item" permission="${permission}" title="${%NewJob(it.newPronoun)}">
+
+    <l:js src="jsbundles/add-item.js" />
+    <l:css src="jsbundles/config-scrollspy.css" />
+        
     <st:include page="sidepanel.jelly" />
     <l:main-panel>
       <j:invokeStatic var="jobs" className="hudson.model.Items" method="all">
         <j:arg type="hudson.model.ItemGroup" value="${it.ownerItemGroup}"/>
       </j:invokeStatic>
-      <n:form nameTitle="${%JobName(it.newPronoun)}" copyTitle="${%CopyExisting(it.newPronoun)}" showCopyOption="${!empty(app.itemMap)}"
-              descriptors="${jobs}" checkUrl="${rootURL}/checkJobName" xmlns:n="/lib/hudson/newFromList" />
+      
+      <j:set var="descriptors" value="${h.filterDescriptors(it,attrs.descriptors)}" />
+      <s:form method="post" action="${attrs.action?:'createItem'}" name="createItem">
+        <div id="j-add-item-form-data" data-root="${rootURL}/"></div>
+
+        <j:if test="${!empty(app.itemMap)}">
```

Comment:
Not a regression in this PR, but BTW this code is incorrect (a Folders compatibility bug).